backend/routes/loginAuth.ts

```diff
@@ -1,26 +1,21 @@
-import { type User } from "@prisma/client";
-import express from "express";
+import { type User } from '@prisma/client';
+import express from 'express';
 import passport from 'passport'
 import prisma from '../utils/prismaClient'
 
 const loginAuthRouter = express.Router()
-const db = prisma;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const db = prisma
 
 loginAuthRouter.get('/password',
-  function (req, res, next) {
-    res.render('login');
-  });
-
-loginAuthRouter.post("/password", passport.authenticate('local'),
   function (req, res) {
-    console.info('user from local auth: ' + (req.user as User).id);
-    res.redirect('/');
-  });
+    res.render('login')
+  })
 
-loginAuthRouter.post("/password", passport.authenticate('local'),
+loginAuthRouter.post('/password', passport.authenticate('local'),
   function (req, res) {
-    console.info('user from local auth: ' + (req.user as User).id);
-    res.redirect('/'); //this is not really working, the redirect is still coming from the client
-  });
+    console.log('user from local auth: ' + (req.user as User).id)
```

Comment:
generally we use `info` logs for informational logs that we actually want to keep around - is this console log something you're using for debugging or should we keep this log in there?

Reply:
yeah it was being used for debugging I'll go ahead and remove.